numba/tests/test_types.py

```diff
@@ -561,15 +561,6 @@ def check_error(sig, msg):
 
 
 class TestRecordDtype(unittest.TestCase):
-    def test_record_type_equiv(self):
```

Comment:
Please could you explain why this test is being removed (I presume something is being deprecated/removed, but would be good to have a record)? Thanks.